digital-koiyu/usr/share/asteroid-launcher/watchfaces/digital-koiyu.qml

```diff
@@ -0,0 +1,435 @@
+                        anchors.fill: parent
+                        source: imgPath + "main-" + (index + 1) + ".svg"
+                        sourceSize.width: width*2
+                        sourceSize.height: height*2
+                        visible: digit.value == 32 ? false : decode.segment16Visible[digit.value-48][index]
+                    }
+                }
+            }
+        }
+        Item {
+            id: dotMatrixRoot
+            width: 172.1*parent.width/320
+            height: 52.2*parent.height/300
+            y: 41.800*parent.width/320
+            x: 76.525*parent.height/300
+            property int columns: 23
+            property int rows: 7
+            Repeater {
+                model: dotMatrixRoot.rows*dotMatrixRoot.columns
+                Rectangle {
+                    height: parent.height*0.95/dotMatrixRoot.rows //these 0.95 add a tiny spacing between pixels
+                    width: parent.width*0.95/dotMatrixRoot.columns
+                    antialiasing: true
+                    property int row: Math.floor(index/dotMatrixRoot.columns)
+                    property int column: index%dotMatrixRoot.columns
+                    x: dotMatrixRoot.width*column/dotMatrixRoot.columns
+                    y: dotMatrixRoot.height*row/dotMatrixRoot.rows
+                    color: "#141414"
+                    visible: dotMatrixTextMode ? decode.dotMatrixVisible[dotMatrixText[Math.floor(column/6)]][row][column%6] : false
+                }
+            }
+        }
+        Repeater {
+            id: bottomDigitRepeater
+            anchors.fill: parent
+            model: ListModel {
+                ListElement {
+                    x: 93.209
+                    y: 229.860
+                }
+                ListElement {
+                    x: 127.690
+                    y: 229.860
+                }
+                ListElement {
+                    x: 173.840
+                    y: 229.860
+                }
+                ListElement {
+                    x: 208.322
+                    y: 229.860
+                }
+            }
+            Item {
+                id: digit
+                width: parent.width
+                height: parent.height
+                x: model.x*parent.width/320
+                y: model.y*parent.height/300
+                property string value: bottomDisplayValue.charCodeAt(index)
+                Repeater {
+                    anchors.fill: parent
+                    model: 7
+                    Image {
+                        anchors.fill: parent
+                        source: imgPath + "bottom-" + (index + 1) + ".svg"
+                        sourceSize.width: width*2
+                        sourceSize.height: height*2
+                        visible: digit.value == 32 ? false : decode.segment7Visible[digit.value-48][index]
+                    }
+                }
+            }
+        }
+        // ColorOverlay{
```

Comment:
In your PR description you ask for ways to re-color the svg, did this not work ?

Reply:
Note that generally speaking we shouldn't add commented code to our version control, either code deserves to be there and it's not commented or it doesn't and it's either not there or in the log. Otherwise these tend to be distracting and get out of sync with the rest of the codebase fairly quickly